pkg/console/consoleserver/config_builder_test.go

```diff
@@ -47,11 +45,7 @@ func TestConsoleServerCLIConfigBuilder(t *testing.T) {
 			name: "Config builder should handle cluster info",
 			input: func() Config {
 				b := &ConsoleServerCLIConfigBuilder{}
-				return b.
-					APIServerURL("https://foobar.com/api").
-					Host("https://foobar.com/host").
-					LogoutURL("https://foobar.com/logout").
-					Config()
+				return Config()
```

Comment:
This is wrong. For some reason GoLand "move" command, which should update file paths only, eliminated code.